packages/api/src/web5.ts

```diff
@@ -152,6 +166,7 @@ export class Web5 {
     this.did = new DidApi({ agent, connectedDid });
     this.dwn = new DwnApi({ agent, connectedDid });
     this.vc = new VcApi({ agent, connectedDid });
+    this.platformAgent = agent as Web5PlatformAgent;
```

Comment:
Since `Web5PlatformAgent` is purely a compile time Type that gets stripped out in production, I don't think any runtime code is necessary to solve for what's being solved for here

Reply:
Yea agreed.  I don't think this is a pattern we'd want to follow and have duplicate properties for the same thing.

Reply:
ACK